packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -713,7 +719,7 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       labelWidth += Math.ceil(longestLabelWidth * Math.cos(Math.PI / 4));
     }
     // Case: truncated labels
-    else if (this.props.showXAxisLablesTooltip) {
+    else if (this.props.showXAxisLablesTooltip && this._tickValues) {
```

Comment:
Shouldn't this whole function be get called in the second render cycle only. I see this used in rotated labels case also.

Reply:
Done